RELEASE.md

```diff
@@ -0,0 +1,44 @@
+## Release version steps
+
+
+**Protocol Version Verification**
+
+-   Ensure the Snyk Language Server Protocol version is correct in the plugin. 
+`ProtocolVersion`  in  `LsConstants`  
+
+
+**Initiate Release**
+
+-   Press the release button to start the release process.
+
+
+**Release Notes**
+
+-   Edit or generate release notes on GitHub.
+It's okay to include all items from any intermediate hotfix releases in the release notes.
+
+
+**Marketplace Availability**
+
+-   Check that the new release appears on all relevant Marketplaces.
```

Comment:
This is too generic in this repo. For Jetbrains, there's a review process, so it will only appear after a positive review.